index.html

```diff
@@ -2135,10 +2135,10 @@ <h2>Assertion</h2>
 
         <p>
 An example of when this property is useful is during the processing of a
-<a>verifiable credential</a> by a verifier. During validation, a verifier checks to
-see if a <a>verifiable credential</a> has been signed by the <a>DID Subject</a> by
-checking that the <a>verification method</a> used to assert the proof is
-associated with the <code><a>assertionMethod</a></code> property in the
+<a>verifiable credential</a> by a verifier. During verification, a verifier
+checks to see if a <a>verifiable credential</a> has been signed by the <a>DID
+Subject</a> by checking that the <a>verification method</a> used to assert the
```

Comment:
Maybe generalize this sentence ("proof" instead of "signature")?

```suggestion
checks to see if a <a>verifiable credential</a> contains a proof created by the <a>DID
Subject</a> by checking that the <a>verification method</a> used to assert the
```

Reply:
+1 @peacekeeper keeping evidence beyond digital signatures alive is a great point. At some point I'll work on adding a LDP suite based on privacy pass to describe how this might work in practice.